graphql/queries/fetch-labels.graphql

```diff
@@ -0,0 +1,12 @@
+query FetchLabels($owner: String!, $name: String!) {
+  repository(owner: $owner, name: $name) {
+    labels(first: 100) {
```

Comment:
Is pagination being used to fetch all labels? e.g. beyond the first 100

Missing `$cursor` I think

Might want to change to this number to a smaller number (e.g. 4) and test if it fetches all labels too

Reply:
thanks! thats a good way to test 👍